drinfomon.lic

```diff
@@ -553,41 +553,43 @@ sleep(0.1) until Char.name && !Char.name.empty?
 CharSettings['bank_accounts'] = {} unless CharSettings['bank_accounts'].class == Hash
 CharSettings['vault_info'] = {} unless CharSettings['vault_info'].class == Hash
 
-kronar_banks = ['Crossings', 'Dirge', 'Leth Deriel', 'Ilaya Taipa']
-lirum_banks = ["Aesry Surlaenis'a", "Hara'jaal", "Mer'Kresh", "Muspar'i", 'Ratha', 'Riverhaven', "Rossman's Landing", 'Therenborough', 'Throne City']
-dokora_banks = ['Ain Ghazal', 'Boar Clan', "Chyolvea Tayeu'a", 'Hibarnhvidar', 'Shard', "Raven's Point"]
+kronar_banks = ['Crossings', 'Dirge', 'Ilaya Taipa', 'Leth Deriel']
+lirum_banks = ["Aesry Surlaenis'a", "Hara'jaal", "Mer'Kresh", "Muspar'i", "Ratha", "Riverhaven", "Rossman's Landing", "Therenborough", "Throne City"]
+dokora_banks = ["Ain Ghazal", "Boar Clan", "Chyolvea Tayeu'a", "Hibarnhvidar", "Fang Cove", "Raven's Point", "Shard"]
```

Comment:
* Sorted entries alphabetically
* Standardized on double-quote rather than some entries with single and some with double